src/goto-programs/json_goto_trace.cpp

```diff
@@ -114,6 +111,13 @@ void convert(
             type_string=from_type(ns, identifier, symbol->type);
 
           json_assignment["mode"]=json_stringt(id2string(symbol->mode));
+          if(step.full_lhs_value.is_not_nil())
+            full_lhs_value=json(step.full_lhs_value, ns, symbol->mode);
```

Comment:
It could happen if the solver we are using returns a `nil_exprt` with a call to get during the production of the trace, but solvers probably shouldn't do that...
So we can either forget about this check and call json (which will give "name": unknown") or add an assertion. I will try to add the assertion and run our tests with `--json-ui` to see if something goes wrong.